airflow/providers/google/cloud/sensors/bigquery.py

```diff
@@ -75,6 +75,8 @@ def __init__(
         gcp_conn_id: str = "google_cloud_default",
         delegate_to: str | None = None,
         impersonation_chain: str | Sequence[str] | None = None,
+        deferrable: bool = False,
+        polling_interval: float = 5.0,
```

Comment:
Should we use `poke_interval` and remove it instead?

Reply:
Yes I think so

Reply:
I just add a deprecation warning to that and assign it to `poke_interval`